### PR TITLE
devices: droidian for lavender, sargo, angelica, dandelion and onclite

### PR DESCRIPTION
**Pull request by FakeShell**
all have been tested confirmed working with the installer

### CODE REVIEW

File: v2/devices/angelica.yml [hunk 1]
```diff
@@ -162,3 +162,56 @@ operating_systems:
...
+    compatible_installer: ">=0.9.2-beta"
+    options:
+      - var: "variant"
+        name: "Variant"
+        tooltip: "The graphical shell to install"
+        type: "select"
+        values:
+          - value: "phosh"
+            label: "Phosh"
+    steps:
```
Comment (NeoTheThird) [inline v2/devices/angelica.yml:176]:
```suggestion
    prerequisites: []
    steps:
```

File: v2/devices/dandelion.yml [hunk 2]
```diff
@@ -161,3 +161,56 @@ operating_systems:
...
+    compatible_installer: ">=0.9.2-beta"
+    options:
+      - var: "variant"
+        name: "Variant"
+        tooltip: "The graphical shell to install"
+        type: "select"
+        values:
+          - value: "phosh"
+            label: "Phosh"
+    steps:
```
Comment (NeoTheThird) [inline v2/devices/dandelion.yml:175]:
```suggestion
    prerequisites: []
    steps:
```

File: v2/devices/lavender.yml [hunk 3]
```diff
@@ -164,3 +164,56 @@ operating_systems:
...
+    compatible_installer: ">=0.9.2-beta"
+    options:
+      - var: "variant"
+        name: "Variant"
+        tooltip: "The graphical shell to install"
+        type: "select"
+        values:
+          - value: "phosh"
+            label: "Phosh"
+    steps:
```
Comment (NeoTheThird) [inline v2/devices/lavender.yml:178]:
```suggestion
    prerequisites: []
    steps:
```

File: v2/devices/onclite.yml [hunk 4]
```diff
@@ -141,3 +141,62 @@ operating_systems:
...
+    compatible_installer: ">=0.9.2-beta"
+    options:
+      - var: "variant"
+        name: "Variant"
+        tooltip: "The graphical shell to install"
+        type: "select"
+        values:
+          - value: "phosh"
+            label: "Phosh"
+    steps:
```
Comment (NeoTheThird) [inline v2/devices/onclite.yml:155]:
```suggestion
    prerequisites: []
    steps:
```

File: v2/devices/sargo.yml [hunk 5]
```diff
@@ -128,3 +128,59 @@ operating_systems:
...
+    compatible_installer: ">=0.9.2-beta"
+    options:
+      - var: "variant"
+        name: "Variant"
+        tooltip: "The graphical shell to install"
+        type: "select"
+        values:
+          - value: "phosh"
+            label: "Phosh"
+    steps:
```
Comment (NeoTheThird) [inline v2/devices/sargo.yml:142]:
```suggestion
    prerequisites: []
    steps:
```

**Review by NeoTheThird — COMMENTED** [review]
hey, sorry for the delay. suggested a small fix you can commit, then, maybe run the formatter and validator locally and we should be good :) thanks for your contribution!

**Comment by FakeShell** [comment]
it should run without any errors now

**Pull request merged**